IFC4x3/Properties/e/ElongationPercentage_0HzBKu8tvAFQLOpm2d1Z0$/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ElongationPercentage_0HzBKu8tvAFQLOpm2d1Z0" Name="ElongationPercentage" UniqueId="11f4b538-237e-4a3d-a558-cf00a706303f" PrimaryDataType="IfcNormalisedRatioMeasure" />
```

Comment:
Was this not supposed to be removed from `IfcCableCarrierSegment` also? I would expect a removal here.